io/FV3GFS_io.F90

```diff
@@ -1647,14 +1648,17 @@ subroutine sfc_prop_restart_read (Sfcprop, Atm_block, Model, fv_domain, warm_sta
           Sfcprop(nb)%sfalb_lnd(ix)       = sfc_var2(i,j,nvar_s2m+28)
           Sfcprop(nb)%sfalb_lnd_bck(ix)   = sfc_var2(i,j,nvar_s2m+29)
           Sfcprop(nb)%sfalb_ice(ix)       = sfc_var2(i,j,nvar_s2m+30)
+          nvar_s2me = nvar_s2m+30
```

Comment:
this doesn't seem correct to me. since you added 18 before, you are only adding 12 new variables

Reply:
fixed.